Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -188,28 +233,19 @@ - (void)showInterstitialAdForParameters:(id<MAAdapterResponseParameters>)paramet
     BOOL isBiddingAd = [bidResponse al_isValidString];
     NSString *placementIdentifier = parameters.thirdPartyAdPlacementIdentifier;
     [self log: @"Showing %@interstitial ad for placement: %@...", ( isBiddingAd ? @"bidding " : @"" ), placementIdentifier];
-    
-    [self.router addShowingAdapter: self];
-    
-    NSError *error;
-    BOOL willShow = NO;
-    if ( isBiddingAd )
+
+    if ( self.vungleInterstitialAd && [self.vungleInterstitialAd canPlayAd] )
     {
-        if ( [[VungleSDK sharedSDK] isAdCachedForPlacementID: placementIdentifier adMarkup: bidResponse] )
+        UIViewController *presentingViewController;
+        if ( ALSdk.versionCode >= 11020199 )
```

Comment:
I saw `11020199` is used twice, might we define it as constant?  Thanks a lot. :)